_data/wgs.yml

```diff
@@ -2,68 +2,70 @@
   description: "What does it take to be an open source leader? How do leaders emerge, and what can we learn about what makes leaders effective? What are the pressures and issues that leaders face, and how can we mitigate them through active development?"
 
 - title: Open Source Funders
   bottomliner: Richard Littauer
   slug: open-source-funders
   status: Incubating
-  ranking: 1
+  ranking: 0
   description: "There's no current database of how much money goes into open source: we want to build this. Follow the money - where does it come from, where does it go, and how useful are different avenues of funding for open source projects at scale."
 
 - title: Sustainability Readiness
   bottomliner: Erin McKean
   slug: sustainability-readiness
   status: Incubating
-  ranking: 1
+  ranking: 0
   description: "How can we ensure that a new open source project is born sustainable? What steps can be put into place to fight entropy in both code and governance structures in open source?"
 
 - title: Licensing
   bottomliner: Benjamin Nickolls
   slug: licensing
   status: Incubating
-  ranking: 1
+  ranking: 0
   description: "What is licensing, anyway? What's the current state of the ecosystem, where are we going, and how can we talk about licenses without flipping over the proverbial dinner table?"
 
 - title: Personas on Sustainability
   bottomliner: Jacob Redding
   slug: personas
   status: Incubating
-  ranking: 1
+  ranking: 0
   description: "Who are sustainers, anyway? What does it mean to sustain a project? How can we use persona development to learn more about the needs of all of the stakeholders in open source?"
 
 - title: 2020 Sustain Summit Report
   bottomliner: Benjamin Nickolls
   slug: 2020-summit-report
   status: Incubating
-  ranking: 1
+  ranking: 0
   description: "It was great, everyone loved it, but - what happened at the 2020 Sustai Summit for those of us who weren't there, or who were busy thinking about a couple of hard problems in open source? How can we further Sustain's goals, going forward?"
 
 - title: Academic and Specialized Projects
   bottomliner: Yo Yehudi
   slug: academic-projects
   status: Incubating
-  ranking: 1
+  ranking: 0
   description: "Open source doesn't just mean web development. How does sustainability relate to academic and specialized projects that have different needs and users?"
 
 - title: Design and UX
   bottomliner: Georgia Bullen
   slug: design-and-ux
   status: Incubating
-  ranking: 1
+  ranking: 0
   description: "How can we make open source design and user experience sustainable, too? What does sustainability mean for code-adjacent territories?"
 
 - title: Docs
   bottomliner: Chris Chinchilla, Pia Mancini
   slug: docs
+  url: true
   status: Incubating
   ranking: 1
   description: "Who writes, uses, and updates the docs? What do sustainable-first docs look like? How can we write better ones?"
 
 - title: Ethics
   bottomliner: Chris Chinchilla
   slug: ethics
+  url: true
```

Comment:
https://sustainoss.org/working-groups/ethics is not ready yet, and should not have been merged. Keep URL as false.